index.js

```diff
@@ -124,14 +125,16 @@ const runPackageManager = (params, buildType, options) => {
     const spinner = new Spinner();
     // Determine whether skin.json was amongst the chosen options
     const skinIncluded = (options.skin || []).some(pluginId => pluginId === 'skin-json');
+    // From 4.10.0 onwards, the skin.json is bundled inside the html5-skin plugin
+    const skinIsBundled = semver.satisfies(params.version, '>=4.10.0');
     // Filter plugin assets and obtain a list all required files and dependencies
     const resources = resourceManager.filterPluginResources(plugins, params.version, buildType, options);
 
     fs.emptyDirSync(config.BUILD_PATH);
 
     resourceManager.downloadResources(resources, mainBuildPath, true)
     .then(() => {
-      if (!skinIncluded) {
+      if (!skinIncluded || skinIsBundled) {
```

Comment:
When `skin.json` is bundled inside `html5-skin.min.js` there is still one absolute url (the loading spinner, I think) that needs to be replaced with a relative url if we want it to use the self-hosted asset. I decided not to replace it in this case because `html5-skin.min.js` is a much larger file, and an error during the replacement process could introduce problems in the code that could be hard to figure out. 

I think a better approach would be to try to use relative urls by default inside `skin.json`.